tests/optimize/test_backtesting_adjust_position.py

```diff
@@ -251,13 +275,21 @@ def custom_entry_price(proposed_rate, **kwargs):
         ]
     res = backtesting._check_trade_exit(trade, row_sl_tgrd, row_sl_tgrd[0].to_pydatetime())
 
+    print("AFTER EXIT")
+    print(f'trade.orders : {trade.orders}')
+    print(f"trade: {trade}")
+
     # TODO count and validate that there is a open entry order
     print(res)
     print(f'exit reason : {res.exit_reason}')
 
+    sl_order = trade.orders[1]
+
     # TODO This should FAIL curently as exits is not triggered if trade.has_open_orders
     assert res
     assert res.exit_reason == ExitType.STOP_LOSS.value
+    assert sl_order.status == 'closed' # TODO why the stoploss order is not filled ?
```

Comment:
@xmatthias  I must validate that the SL order is properly filled when there is no open entry or exit order, but the SL order is not filled in that case

Reply:
I think your approach is a bit off if i'm honest.

you're calling backtesting internals (`_check_trade_exit()` is not supposed to close the order) - but expect things that happen in other parts of backtesting to also happen.
You CAN make these happen - but it'll not happen automatically if you call partials.

The above called method is just checking, and eventually creating an order.
Handling of the actual order handles after this call - directly in `backtest_loop()`.
That'll handle both orders that were created in this iteration - as well as orders that are there from previous candles and didn't fill yet.

Reply:
Great thank you for the clarification

Reply:
So from what I understand I must do a test strategy that is closer to "test_backtest_one" using "UNITTEST_BTC" data, right?

Reply:
"probably" yes ...